pkg/agent/flowexporter/connections/connections.go

```diff
@@ -108,9 +108,6 @@ func (cs *connectionStore) fillPodInfo(conn *flowexporter.Connection) {
 
 	srcPod, srcFound := cs.podStore.GetPodByIPAndTime(srcIP, conn.StartTime)
 	dstPod, dstFound := cs.podStore.GetPodByIPAndTime(dstIP, conn.StartTime)
-	if !srcFound && !dstFound {
-		klog.Warningf("Cannot map any of the IP %s or %s to a local Pod", srcIP, dstIP)
```

Comment:
@dreamtalen I guess originally this was logged for a lot of Egress traffic? Such connections are dropped later in `exportConn`, but would it make sense to simply omit the connection from the connection store? Or is there some other "legitimate" scenario for which we still need to export the connection?

Reply:
Hi Antonin, I also believe there is no such scenario.
Discussed with @yuntanghsu, we can skip adding this kinds of connection to connection store when it arrives.

Reply:
I've moved this part from fillPodInfo to AddOrUpdateConn. And we won't add the connection to connection map or expirePriorityQueue if we can't find the srcPod info and dstPod info.

Reply:
Yes, for Egress traffic we only export one record at the source Node, and the Egress Node should not export a record for this flow.